### DISCUSSION TITLE
Welcome to github_tut Discussions!

### DISCUSSION THREAD

**Discussion by joechea-aupp** [joechea-aupp/github_tut#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by joechea-aupp** [comment]
fork the repository and do a pull request

**Comment by joechea-aupp** [comment]
resolve the issue mentioned above

**Reply by SanPiseth-beep** [reply]
am cool

**Comment by joechea-aupp** [comment]
dup
